plugins/autocomplete/plugin.js

```diff
@@ -155,8 +155,18 @@
 	 * @param {CKEDITOR.plugins.autocomplete.model.item[]} dataCallback.callback.data The suggestion data that should be
 	 * displayed in the autocomplete view for a given query. The data items should implement the
 	 * {@link CKEDITOR.plugins.autocomplete.model.item} interface.
+	 * @param {String} viewTemplate Changes the view template.
```

Comment:
Also instead of duplicating the description it's good to just put minimal information, like: Template for list item in a dropdown. See `{@<original property>}` for more information.

Pretty much like you did with `outputTemplate`.

Reply:
By original property I mean property in `autocomplete.view` type.